src/fides/api/ops/models/privacy_request.py

```diff
@@ -767,7 +767,7 @@ class ProvidedIdentityType(EnumType):
 
     email = "email"
     phone_number = "phone_number"
-    user_id = "user_id"
+    ga_client_id = "ga_client_id"
```

Comment:
one more question, are we sure that it is the `client_id` and not the `app_instance_id`?

My options in the API request are:
Type of user. One of APP_INSTANCE_ID, CLIENT_ID or USER_ID.

Reply:
oh! I know that the cookie only has the `client_id`. I'm guessing the `app_instance_id` might be the GA value customers have to put in their javascript, but let me take a look at the docs and get back to you.

Reply:
okay great 👍

Reply:
okay yes, looking at this, https://developers.google.com/analytics/devguides/config/userdeletion/v3, it _should_ be `client_id` we want here

Reply:
excellent